modules/ROOT/pages/configure-acm.adoc

```diff
@@ -153,8 +153,8 @@ NOTE: If your Anypoint Platform organization is hosted in the EU, change `+https
 . In *Setup*, search for *Auth. Providers* using the *Quick Find* box and click *Auth. Providers* -> *New*.
 . In *Provider type* select *Open ID Connect*.
 . Set the *Name* to `Anypoint`.
-. Set the *Consumer Key* to the application ID of the Connect App created in Anypoint.
-. Set the *Consumer Secret* to the application secret of the Connect App created in Anypoint.
+. Set the *Consumer Key* to the application ID of the connected app created in Anypoint. This ID was returned by the UI at the end of the step <<create-a-connected-app-with-the-user-interface>> or returned by the API call at the end of the step <<create-a-connected-app-with-an-api-call>>.
```

Comment:
GLOBALLY limit steps, list items, and table cell text to one sentence or fragment, to achieve the intended scannability. If necessary, add details below.
```suggestion
. Set Consumer Key to the application ID of the connected app created in Anypoint. 
This ID was returned by the UI at the end of the step <<create-a-connected-app-with-the-user-interface>> or returned by the API call at the end of the step <<create-a-connected-app-with-an-api-call>>.
```
Also...returned by "Anypoint <what>"?

Reply:
Do these steps have numbers that we can refer to instead of texts?
Are the steps directly above or are they in other sections...or even topics?

Reply:
Much better! Thank you, Kim.